hack/start-operator.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# arguments: IMG, TARGET, (optional) OPERATOR_PATH
+for ARGUMENT in "$@"
+do
+   KEY=$(echo $ARGUMENT | cut -f1 -d=)
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+   export "$KEY"="$VALUE"
+done
+
+if [[ -z $IMG ]]; then
+    echo "Error: IMG is required"
+    exit 1
+fi
+
+if [[ -z $TARGET ]]; then
+    echo "Error: TARGET is required"
+    exit 1
+fi
+
+if [[ $# == 3 ]]; then
+  cd $OPERATOR_PATH
+fi
+
+IMG=$IMG make docker-build docker-push
```

Comment:
If the values are exported at line 9, is IMG=$IMG and TARGET=$TARGET needed?

Reply:
No, fixed